_pytest/assertion/__init__.py

```diff
@@ -113,7 +115,8 @@ def callbinrepr(op, left, right):
         for new_expl in hook_result:
             if new_expl:
                 if (sum(len(p) for p in new_expl[1:]) > 80*8
-                        and item.config.option.verbose < 2):
```

Comment:
IMO, if anything (and this is nitpick), I would move the `sum(len(p) for p in new_expl[1:]) > 80*8` expression into a local variable `expression_too_long` for legibility... but as I said, really minor and doesn't block this PR. :grin: